spec/models/generic_work_spec.rb

```diff
@@ -59,6 +59,39 @@
     end
   end
 
+  describe "embargo" do
+    subject(:work) { described_class.new(title: ['a title'], embargo_release_date: embargo_release_date) }
+    let(:embargo_release_date) { Time.zone.today + 10 }
+
+    it { is_expected.to be_valid }
+
+    context 'with a past date' do
+      let(:embargo_release_date) { Time.zone.today - 10 }
+
+      it { is_expected.not_to be_valid }
+
+      it 'has errors related to the date' do
+        expect { work.valid? }
+          .to change { work.errors.to_a }
+          .from(be_empty)
+          .to include("Embargo release date Must be a future date")
+      end
+    end
+
+    context 'with a saved embargo' do
+      let(:past) { Time.zone.today - 10 }
+
+      before { work.save! }
+
+      it 'can update the embargo with any date' do
+        work.embargo_release_date = past
+
+        expect(work).to be_valid
+        expect{ work.save! }.not_to raise_error
```

Comment:
```suggestion
        expect { work.save! }.not_to raise_error
```

Reply:
Is this requiring a space? Syntax wise, the space makes sense because { = do and if you write it out, it would need spaces. Im on board with this suggestion.